index.html

```diff
@@ -9,12 +9,27 @@
 </head>
 <body>
   <section id="wrapper">
-    <h1>All awesome books</h1>
-    <ul id="book-list"></ul>
-    <div id="divider"></div>
-    <!-- Add book to list -->
-    <form id="form">
-      <h2>Add a new book</h2>
+    <!-- Navigation -->
+    <nav id="navigation">
+      <a href="#" class="nav-logo">Awesome Books</a>
+
+      <ul id="nav-list">
+        <li class="nav-item" id="list"><a href="#" type="button">List</a></li>
+        <li class="nav-item" id="add"><a href="#" type="button">Add new</a></li>
+        <li class="nav-item" id="contact-link"><a href="#" type="button">Contact</a></li>
+      </ul>
+    </nav>
+
+    <!-- Page Title -->
+    <h1 class="page-title">All awesome books</h1>
+
+    <!-- Book List -->
+    <section id="bookSection">
+      <ul id="book-list"></ul>
+    </section>
+
+    <!--Form to add book to list -->
+    <form id="form" class="remove-section">
       <div class="form-input">
         <label for="title" hidden>Title</label>
         <input type="text" id="title" name="title" placeholder="Title">
```

Comment:
- [ ] As it is in the project wireframe, kindly ensure that you include the Date and time in the UI of your project. This way you'll have met all the project requirements.

![image](https://user-images.githubusercontent.com/86318284/191744498-c74da180-9c28-4c3a-acef-ce24b2b91a76.png)